plugin-build/plugin/src/main/java/io/github/wzieba/tracks/plugin/BuildData.kt

```diff
@@ -19,6 +19,7 @@ data class BuildData(
     val taskStatistics: TaskStatistics,
     val buildDataCollectionOverhead: Long,
     val includedBuildsNames: List<String>,
+    val architecture: String,
```

Comment:
I did not added an `enum class` for that as I'm not 100% sure how different computers will return `value of `uname -m`. They should have `arm64` or `x86_64` but I can't be sure.

Eventually, enum here doesn't bring a lot of value either way IMO.